src/uuid_worker.py

```diff
@@ -860,6 +860,8 @@ def getIdInfo(self, app_id):
                 for item in results:
                     if item['submission_id'] == None:
                         item.pop('submission_id')
+                    if item['ancestor_ids'] == None:
```

Comment:
@kburke Not clear- how are `item`s created- is it possible that `'ancestor_id'` doesn't exist at all in the dict?- if so will that cause line 863 to fail.  Also, is it possible for `'ancestor_d`` to be an empty array?-- In which case it won't match the `== None` comparison.